localization/english/wc_holy_orders_l_english.yml

```diff
@@ -2,6 +2,246 @@
+ holy_order_holy_knights_article:0 "the "
+ holy_order_holy_knights:0 "Holy Knights of [TITLE.GetNameNoTierNoTooltip]"
+
+ ### Lizard Group ###
+ holy_order_evolutive_warriors_article:0 "the "
+ holy_order_evolutive_warriors:0 "Evolutive Warriors"
+
+ ### Loa Group ###
+ holy_order_spirit_warriors_article:0 "the "
+ holy_order_spirit_warriors:0 "Spirit Warriors of [FAITH.HighGodName]"
+
+ ### Naarism ###
+ holy_order_army_of_the_light_article:0 "the "
+ holy_order_army_of_the_light:0 "Army of the Light"
+ holy_order_rightful_truth_article:0 "the "
+ holy_order_rightful_truth:0 "Rightful Truth"
+ holy_order_lightforged_article:0 "the "
+ holy_order_lightforged:0 "Lightforged of [FAITH.KnowledgeGodName]"
+
+ ### Necromancy Group ###
+ holy_order_immortal_army_article:0 "the "
+ holy_order_immortal_army:0 "Immortal Army"
+ holy_order_last_breath_article:0 "the "
+ holy_order_last_breath:0 "Last Breath"
+
+ ### Pandaren ###
+ holy_order_white_lotus_article:0 "the "
+ holy_order_white_lotus:0 "White Lotus"
+ holy_order_raging_tiger_article:0 "the "
+ holy_order_raging_tiger:0 "Raging Tiger"
+ holy_order_graceful_crane_article:0 "the "
+ holy_order_graceful_crane:0 "Graceful Crane"
+ holy_order_silent_serpent_article:0 "the "
+ holy_order_silent_serpent:0 "Silent Serpent"
+ holy_order_stubborn_ox_article:0 "the "
+ holy_order_stubborn_ox:0 "Stubborn Ox"
+ holy_order_pandaren_article:0 "the "
+ holy_order_pandaren:0 "Warriors of the Panda Empire of [FAITH.KnowledgeGodName]"
+
+ ### Secular ###
+
+ ### Shamans ###
+ holy_order_earthern_ring_article:0 "the "
+ holy_order_earthern_ring:0 "Earthern Ring"
+ holy_order_earth_wind_and_fire_article:0 "the "
+ holy_order_earth_wind_and_fire:0 "Earth, Wind and Fire"
+
+ ### Shathgral
+ holy_order_twilight_hammer_cult_article:0 "the "
+ holy_order_twilight_hammer_cult:0 "Twilight's Hammer Cult"
+ holy_order_fanatics_article:0 "the "
+ holy_order_fanatics:0 "Fanatics of [FAITH.HighGodName]"
+
+ ### Titanic ###
+# holy_order_faithful_of_tyr_article:0 "the "
+# holy_order_faithful_of_tyr:0 "Faithfuls of Tyr"
+# holy_order_odins_valkyries_article:0 "the "
+# holy_order_odins_valkyries:0 "Odyn's Valkyries"
+# holy_order_chosen_of_freyja_article:0 "the "
+# holy_order_chosen_of_freyja:0 "Chosen of Freya"
+ holy_order_warriors_from_stars_article:0 "the "
+ holy_order_warriors_from_stars:0 "Warriors from the Stars"
+
+ ### Void ###
+ holy_order_forces_of_darkness_article:0 "the "
+ holy_order_forces_of_darkness:0 "Forces of Darkness"
+ holy_order_shadow_of_creation_article:0 "the "
+ holy_order_shadow_of_creation:0 "Shadow of Creation"
+ holy_order_messengers_of_void_lords_article:0 "the "
+ holy_order_messengers_of_void_lords:0 "Messengers of the Void Lords"
+
+ ### Generic ###
+ holy_order_shamanism_article:0 "the "
+ holy_order_shamanism:0 "Shamans of [FAITH.HighGodName]"
+ holy_order_arcane_article:0 "the "
+ holy_order_arcane:0 "Mages of [FAITH.HighGodName]"
```

Comment:
The same with this name.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/126021621-776c773d-d51c-44ed-a446-9691f7701d3e.png)

</details>

Reply:
These looked horrible, changed a lot of names so they sound way better.